chk/modules/http/presentation.py

```diff
@@ -1,25 +1,56 @@
 """
 Presentation related logics
 """
+from typing import Union
 
+from chk.infrastructure.file_loader import FileContext
 
-def make_displayable(response: dict[str, object]) -> str:
-    def headers(res) -> str:
-        """Headers"""
-        return '{}'.format(
-            '\r\n'.join('{}: {}'.format(k, v) for k, v in res.get('headers').items()),
-        )
-
-    if response.get('have_all'):
-        summary = '{} {} {}'.format(
-            response.get('version'),
-            response.get('code'),
-            response.get('reason'),
-        )
-
-        return '{}\r\n{}\r\n\r\n{}'.format(summary, headers(response), response.get('body'))
-    else:
-        response.pop('have_all')
-        for _, val in response.items():
-            if val:
-                return str(val)
+
+class Presentation:
+    """Handle presentation of the outputs of chkware commands."""
+
+    @classmethod
+    def present_result(cls, file_ctx: FileContext, data: Union[dict, BaseException]):
+        """Shows result of execution."""
+        if isinstance(data, dict):
+            print(cls.displayable_summary(file_ctx.filepath, 'Success'))
+            print(cls.displayable_result(data))
+        if isinstance(data, BaseException):
+            print(cls.displayable_summary(file_ctx.filepath, 'Failed'))
+            print(cls.displayable_error(data))
+
+    @classmethod
+    def displayable_result(cls, response: dict[str, object]) -> str:
+        """Return result in presentable format."""
+        prefix = '[RESULT]'
```

Comment:
`'[RESULT]'` may be we don't need this as well?

Reply:
I agree